index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
```

Comment:
```suggestion
<html lang="ru" dir="ltr">
```

сайт у тебя же на русском